content/en/blog/2023/announcing-notation-rc6.md

```diff
@@ -13,7 +13,7 @@ The Notation maintainers are pleased to announce the release of Notation v1.0.0-
 
 This release adds the following major changes:
 
-- Security advisory fixes
+- Security advisory fixes: CVE-2023-33957, CVE-2023-33958, CVE-2023-33959
```

Comment:
Shall we add a link to each advisory? Otherwise, people are not able to find the source.